app-i18n/libime/libime-1.1.3.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Fcitx5 Next generation of fcitx "
+HOMEPAGE="https://fcitx-im.org/"
+SRC_URI="https://download.fcitx-im.org/fcitx5/libime/libime-${PV}_dict.tar.xz"
+
+LICENSE="LGPL-2+"
+SLOT="5"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=app-i18n/fcitx-5.1.5:5
+	app-arch/zstd:=
+	dev-libs/boost:=
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	kde-frameworks/extra-cmake-modules:0
+	virtual/pkgconfig
+	doc? (
+		app-doc/doxygen
+		dev-texlive/texlive-fontutils
+	)
+"
+
+PATCHES="
+	${FILESDIR}/${P}-use-c++11-for-kenlm.patch
+	${FILESDIR}/${P}-fix-the-nanf-value-issue-on-musl.patch
+"
```

Comment:
suggest array style
PATCHES=(
	  "${FILESDIR}/${P}-use-c++11-for-kenlm.patch"
	  "${FILESDIR}/${P}-fix-the-nanf-value-issue-on-musl.patch"
)